test/integration/index.js

```diff
@@ -1,26 +0,0 @@
-// Copyright (c) 2015 Uber Technologies, Inc.
```

Comment:
Consider keeping an `index.js` file.

There is value to being able to run `node test/integration/` and run just the integration tests.

Alternatively I guess you can run `tape test/integration/*.js`, I've always been a fan of "tests are just a node program" approach.